security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -74,13 +83,13 @@ const (
 	notifyK8sSecretTimeout = 30 * time.Second
 
 	// The well-known path for an existing certificate chain file
-	existingCertChainFile = "./etc/certs/cert-chain.pem"
+	CertChainFilePath = "./etc/certs/cert-chain.pem"
```

Comment:
Start with lowercases, and name as "defaultCertChainFilePath"? Same for the other two.

Reply:
Done, thanks.